README.md

```diff
@@ -254,7 +254,57 @@ reactor.listenTCP(8000, factory)
 reactor.run()
 ```
 
-#### WSGI
+#### Flask
+
+To use prometheus with [flask](http://flask.pocoo.org/) we need to serve metrics through a promethues wsgi application. This can be achieved using [flask's application dispatching|http://flask.pocoo.org/docs/latest/patterns/appdispatch/]. Below is a working example.
+
+```python
+import time
+from flask import request, Flask
+from werkzeug.wsgi import DispatcherMiddleware
+from prometheus_client import Counter, Histogram, make_wsgi_app
+
+
+# Create my app
+app = Flask(__name__)
+
+# Add prometheus wsgi middleware to route /metrics requests
+app_dispatch = DispatcherMiddleware(app, {
+    '/metrics': make_wsgi_app()
+})
+
+@app.route('/')
+def hello():
+    return "Hello world !!"
+
+# Expose some basic metrics of the application
+REQUEST_LATENCY = Histogram('request_latency_seconds', 'API Request Latency', ['method', 'endpoint'])
```

Comment:
Keep this minimal, just enough to expose metrics at all like in the other examples.

Reply:
Done